walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,339 @@
+:showcase-app-link: https://github.com/aerogear/ionic-showcase
```

Comment:
This will be the main one to focus on - for the moment we can ignore the one above. 
The difference is that above one:

- do not configure AMQ for production usage.
- do not use mdc for config (just ask people to add url manually)
- do not configure audit logging and keycloak